packages/components/src/input-control/stories/index.story.tsx

```diff
@@ -16,7 +16,8 @@ import { InputControlSuffixWrapper } from '../input-suffix-wrapper';
 import Button from '../../button';
 
 const meta: Meta< typeof InputControl > = {
-	title: 'Components (Experimental)/InputControl',
+	title: 'Components/Selection & Input/Common/InputControl',
```

Comment:
Changed from `experimental` to `stable` as per the sitemap. If not this can be reverted to use `experimental` only.

Reply:
Right, we can't just move components under the Experimental grouping because there are other administrative things that need to be done first. Let's revert for now, and we'll move the groupings for these ones once they are properly stabilized (#59418).

Reply:
Thanks @mirka, I would update the PR.